salt/transport/ipc.py

```diff
@@ -175,13 +175,25 @@ def return_message(msg):
                     body = framed_msg['body']
                     self.io_loop.spawn_callback(self.payload_handler, body, write_callback(stream, framed_msg['head']))
             except tornado.iostream.StreamClosedError:
-                log.trace('Client disconnected from IPC {0}'.format(self.socket_path))
+                log.trace('Client disconnected '
+                          'from IPC {0}'.format(self.socket_path))
                 break
+            except socket.error as e:
```

Comment:
A style nitpick, but I would suggest using `exc` instead of `e` to be consistent with the surrounding code. The Linter might complain about this name being too short, but I am not sure if it will.